.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: [
+    'plugin:vue/essential',
+    '@vue/airbnb',
```

Comment:
I dont like this

Can we use Standard  ???? :pizza:

Reply:
Show me the code. Do you mean without "airbnb" linter?

Reply:
Change airbnb in favor or standardjs :100: 

https://standardjs.com/